src/Config.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+/* Name of the bootloader. This name is copied to the
+ * status flash at first boot */
+const char* const BOOTLOADER_NAME = "Okra Bootloader";
```

Comment:
Yeah, we can simply change the name to something like "Okra Bootloader v0.1.2"

Reply:
Cool lets leave it for now. 
When we figure out the versioning for Edamame we can also inject a version number here, on build. It might be helpful for Edamame to read and print bootloader version on startup.